### Concept
This package has been written to help you: 
1. generate a psudocontig starting from a draft genome (with Contiguator) -- it uses a reference genome as a template
2. do the variant calling against a reference genome (with mummer) -- it generates a .vrt file

### Dependencies
* contiguator (NOTE: contiguator should be in the $PATH variable. This means that you should be able to type ./Contiguator.py and execute the command)
* mummer
* python (2.7) 

### Usage
1. First of all we need to create the Pseudocontig
```
vrtTools-generatePseudocontig.py example/H37Rv_mod.fasta example/00-R0025_denovo.fasta

```
CONTIGuator will create the folders Map_<id_reference> and UnMappedContigs!


NOTE: CONTIGuator is very picky about the identifiers inside your fasta file. In order to avoid problems, please remove all dots present in the identifier before running vrtTools-generatePseudocontig.py. A command that can help you this this is:
```bash
cat H37Rv.fasta |sed -e 's/\./_/' > H37Rv_mod.fasta
```

2. We can now align the reference and the Pseudocontig with mummer
```bash
vrtTools-runMummer.py example/H37Rv_mod.fasta Map_NC.000962.3/PseudoContig.fsa out_mummer
```

3. We can now convert the .snps file generated by mummer into a .vrt file
```bash
vrtTools-snps2vrt.py out_mummer.snps example/H37Rv_mod.fasta Map_NC.000962.3/PseudoContig.fsa ref_vs_00-R0025.vrt
```


