###Concept
This package has been written to help you: 
(1) generate a psudocontig starting from a draft genome (with Contiguator) -- it uses a reference genome as a template
(2) do the variant calling against a reference genome (with mummer) -- it generates a .vrt file

###Dependencies:
*contiguator (NOTE: contiguator should be in the $PATH variable. This means that you should be able to type ./Contiguator.py and execute the command)
*mummer
*python (2.7) 

###Usage:
1. create a new empty directory and go into it
    mkdir test
    cd test

2. First of all we need to create the Pseudocontig
    vrtTools-generatePseudocontig.py example/H37Rv_mod.fasta example/00-R0025.fasta 

CONTIGuator will create the folders Map_<id_reference> and UnMappedContigs


3. We can now align the reference and the Pseudocontig with mummer
    vrtTools-runMummer.py example/H37Rv_mod.fasta Map_NC.000962.3/PseudoContig.fsa out_mummer


4. We can now convert the .snps file generated by mummer into a .vrt file
    vrtTools-snps2vrt.py out_mummer.snps example/H37Rv_mod.fasta Map_NC.000962.3/PseudoContig.fsa ref_vs_00-R0025.vrt



